data_models/aws_alb_data_model.yml

```diff
@@ -0,0 +1,12 @@
+AnalysisType: datamodel
+LogTypes: 
+  - AWS.ALB
+DataModelID: AWS.ALB.DataModel
```

Comment:
Is it necessary to add DataModel at the end? Shouldn't there be a 1:1 mapping between the LogType and the DataModelID?

Reply:
Or does the API allow for multiple by the same log type? How does it handle it in that case?

Reply:
Ahh, good point - that does make it a bit redundant in the ID. 

It currently only allows one log type per data model - but was considering potentially supporting multiple LogTypes per DataModel in the future. The ID field is required -  Do you think we should just remove it / use LogType as an "ID" to make this more concise?

Reply:
I guess another reason to have an ID that isn't just the LogType is... if a user wants to define their own DataModel for a LogType where there is also a panther-managed DataModel.  They can disable the panther-managed one and add their own (or I suppose they could delete the panther-managed one as well)

Reply:
> supporting multiple LogTypes per DataModel in the future.

How could this work? What if there's overlap on the keys?

> Do you think we should just remove it / use LogType as an "ID" to make this more concise?

Yeah, that makes sense to me. I'm imaging something like: `LogType: AWS.ALB`

> They can disable the panther-managed one and add their own 

I'd say just modifying the existing one would be a little easier on the user's part

Reply:
> Yeah, that makes sense to me. I'm imaging something like: LogType: AWS.ALB

> I'd say just modifying the existing one would be a little easier on the user's part

This changes the existing BE work and potentially the [FE](https://github.com/panther-labs/panther/issues/1974) work going on right [now](https://github.com/panther-labs/panther/issues/2072) as well (to: (1) remove ID and name fields (2) change LogType from array to str) Is that something that should be done now / this sprint? cc: @mattthetraveler

Reply:
@lindsey-w will you be able to contain the change in this sprint?

FYI @3nvi for the work you're doing on Data Models, might be slight change

Reply:
Changing LogType from array of str to str I think I could definitely complete this sprint.  I think changing what is used as the ID might require some additional work/thought - I might also like to have a larger discussion about that if possible.

Reply:
I'd rather leave the ID in, although we can drop the `.Datamodel` suffix as it's a bit redundant. Everything else has an ID, and I think we could definitely support multiple data models per logtype in the future even if we don't know exactly how right now. An order of precedence in the datamodels, or a data model type, or some fancy merging where overlapping fields is "undefined behavior" or something. I don't see any benefit in removing the ID field, and I see potential downsides in making future functionality more difficult to implement. I could also foresee issues with the new pack functionality if users can't disable data models and create new ones with different IDs.

And definitely we should let one data model apply to multiple log types. I'm sure the Crowdstrike log types will benefit from this. Similarly the syslog log types. And we've had several customers put in multiple custom log schemas that are identical except for one field.

I'd say leave it as it is today, except to possibly drop the `.Datamodel` suffix for conciseness. Maybe replace it with something like `.Core` or `.Default` so customers can differentiate it from their own data models.